### ISSUE TITLE
Program crashes when switching audio devices in Windows

### ISSUE THREAD

**Issue by YourDesiredUsername** [stepmania/stepmania#849]
I switch the active audio device in Windows between my speakers & TV as needed.  Switching while Stepmania is running can make it crash.

Crashinfo:  http://pastebin.com/9v4SpPWn

Log (embarrassingly huge song list edited out):  http://pastebin.com/NHr8Ann3

**Comment by SpoOkyMagician** [comment]
Yeah, I have had this happen before as well with my headset/speakers.
